libcaf_core/caf/flow/observable.hpp

```diff
@@ -635,6 +640,12 @@ transformation<step::on_error_complete<T>> observable<T>::on_error_complete() {
   return transform(step::on_error_complete<T>{});
 }
 
+template <class T>
+transformation<step::on_error_return_item<T>>
+observable<T>::on_error_return_item(T&& t) {
+  return transform(step::on_error_return_item<T>{std::forward<T>(t)});
+}
```

Comment:
Same here.

```suggestion
transformation<step::on_error_return_item<T>>
observable<T>::on_error_return_item(T item) {
  return transform(step::on_error_return_item<T>{std::move(item)});
}
```

(plus adjusting the definition)